__tests__/groundplaces/autocomplete.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { GroundPlacesController } from '../../src/classes/groundplaces';
+import * as largeGroundPlacesFile from '../../mocks/largeGroundPlacesFile.json';
+import { AutoComplete, AutoCompleteFilters, GroundPlaces } from '../../src/types';
+
+const { STOP_GROUP, STOP_CLUSTER, SEGMENT_PROVIDER_STOP } = AutoCompleteFilters;
```

Comment:
Prefer you to use `AutoCompleteFilters.STOP_GROUP` your code will be more readable

Reply:
Ok done